custom/icds_reports/templatetags/icds_format.py

```diff
@@ -39,8 +48,8 @@ def icds_toilet_type(value):
 @register.filter(name='icds_material_available')
 def icds_material_available(data):
     if data:
-        kit_available = data['preschool_kit_available']
-        kit_usable = data['preschool_kit_usable']
+        kit_available = data['preschool_kit_available'] if 'preschool_kit_available' in data else False
```

Comment:
This could be `data.get('preschool_kit_available')` which would return `None` if it doesn't exist in the dictionary. which is falsy so it should work for the following conditional in line 53